opencontrol.yaml

```diff
@@ -0,0 +1,33 @@
+schema_version: "1.0.0"
+name: process-controls
+metadata:
+  description: Template Process/Procedural Controls for an Information System
+  maintainers:
+    - Shawn Wells <shawn@redhat.com>
```

Comment:
Change this to us

Reply:
Actually - I would put this as some generic user since this is a template right?

As Tibor would do:

`- Gen Eric <generic@acme.com>`

Reply:
crap... did it in another file. Just a sec